### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by akruphi**
* Minor after #2542

* Yet more customization via menu/dialogs after #2452
    
    изображение

    
    изображение

### CODE REVIEW

File: far2l/src/cfg/config.cpp [hunk 1]
```diff
@@ -380,6 +386,9 @@ static LONG_PTR WINAPI DirCfgDlgProc(HANDLE hDlg, int Msg, int Param1, LONG_PTR
 		SendDlgMessage(hDlg, DM_LISTSET, ID_DIRCFG_STYLE_COMBO, (LONG_PTR)&farlist);
 //		SendDlgMessage(hDlg, DM_LISTSETCURPOS, dircfg_data->DirNameStyleComboID, (LONG_PTR)&flpos);
 		update_surrcombo( );
+
+		swprintf(tmp, 48, L"\"%ls\"", SymLinkNames[dircfg_data->DirNameStyle].CPtr());
```
Comment (elfmz) [inline far2l/src/cfg/config.cpp:390]: please use ARRAYSIZE(tmp) instead of hardcoded 48 (btw its 48 * 4)

File: far2l/src/cfg/config.cpp [hunk 2]
```diff
@@ -388,6 +397,8 @@ static LONG_PTR WINAPI DirCfgDlgProc(HANDLE hDlg, int Msg, int Param1, LONG_PTR
 
 			dircfg_data->DirNameStyle = SendDlgMessage(hDlg, DM_LISTGETCURPOS, ID_DIRCFG_STYLE_COMBO, (LONG_PTR)0);
 			update_surrcombo( );
+			swprintf(tmp, 48, L"\"%ls\"", SymLinkNames[dircfg_data->DirNameStyle].CPtr());
```
Comment (elfmz) [inline far2l/src/cfg/config.cpp:400]: please use ARRAYSIZE(tmp) instead of hardcoded 48 (btw its 48 * 4)

File: far2l/src/cfg/config.cpp (hunk 1 above)
Comment (akruphi) [reply far2l/src/cfg/config.cpp]: Done

File: far2l/src/cfg/config.cpp (hunk 2 above)
Comment (akruphi) [reply far2l/src/cfg/config.cpp]: done

**Pull request merged**